src/main/java/com/google/sps/servlets/FetchQuestionsServlet.java

```diff
@@ -35,18 +35,30 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** 
- * This servlet will retrieve forum posts to be displayed on the page.
+ * Retrieves questions to be displayed on the page.
  */
-@WebServlet("/fetch-forum")
-public class FetchForumServlet extends HttpServlet {
+@WebServlet("/fetch-questions")
+public class FetchQuestionsServlet extends HttpServlet {
 
   /** 
-   * This method will get the forum questions from the query and return them as a JSON string.
+   * Gets  the questions from the query and return them as a JSON string.
    */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     List<QuestionObject> questions = new ArrayList<>();
-    String query = Utility.fetchQuestionQuery;
+    
+    String query;
+
+    // ID of the question to query. -1 means that all questions are to be queried.
```

Comment:
nit: You probably don't need the -1 comment anymore because you are now using SqlConstants. Another developer probably won't know/care what FETCH_ALL_QUESTIONS literally equals. That's the beauty of these consts - they're so readable :)

Reply:
nice! done